package.json

```diff
@@ -8,24 +8,57 @@
     "test": "jest"
```

Comment:
Matt, I'm sooo glad you were able to figure the setup for the tests! I spent so much time this morning trying to figure it out, but nothing worked. 
Maybe we could ask the instructors if we need all of these dependencies. We could also ask about the test in the Home component (regarding the Map). 
Nice job :)